flow_screen_components/quickChoiceFSCproject/force-app/main/default/lwc/quickChoiceFSC/quickChoiceFSC.js-meta.xml

```diff
@@ -11,13 +11,16 @@
             <property name="value" label="Value" type="String"></property>
             <property name="choiceValues" label="Choice Values (String Collection)" type="String[]"></property>
             <property name="choiceLabels" label="Choice Labels (String Collection)" type="String[]"></property>
+            <property name="choiceIcons" label="Choice Icons (String Collection)" type="String[]"></property>
+            <property name="includeIcons" label="Include Icons in Text Box?" type="Boolean"></property>
```

Comment:
always look for the most evocative language when naming variables. In this case, showing vs not showing is clearer than including vs not including, no?

Reply:
I've changed the variable to showIcons and updated the documentation

Reply:
because of this change and the managed package i'm unable to deploy these changes to hostquick-dev

Reply:
if we revert this change, can you then deploy to that org and update the existing managed package?

Reply:
yes

Reply:
ok go for it

Reply:
See new pull request #237